insights/parsers/filefrag.py

```diff
@@ -34,7 +34,9 @@ def parse_content(self, content):
         if len(content) == 0 or (len(content) == 1 and 'No such file or directory' in content[0]):
             raise SkipComponent("Error: ", content[0] if content else 'empty file')
         for line in content:
-            file = line.split()[0].strip(':')
-            extents = line.split()[1]
-            if extents.isdigit():
-                self[file] = int(extents)
+            split_list = line.split()
+            if (not len(split_list) < 2) or 'No such file or directory' in line:
+                file = split_list[0].strip(':')
+                extents = split_list[1]
+                if extents.isdigit():
+                    self[file] = int(extents)
```

Comment:
What about adding a `self.unparsed_lines = []` to store the issue lines, just in case for further usage?